src/file-picker/src/FilePicker.js

```diff
@@ -182,7 +186,12 @@ FilePicker.propTypes = {
   /**
    * Placeholder of the text input
    */
-  placeholder: PropTypes.string
+  placeholder: PropTypes.string,
+
+  /**
+   * Class name passed to the FilePicker.
```

Comment:
Can you add "Only use this if you know what you are doing." to these comments

Reply:
@tusharf5 sorry just noticed, that we want to add this

Reply:
Sure, I'll add it just now.